src/packet/BinaryWriter.js

```diff
@@ -1,137 +1,62 @@
-﻿'use strict';
+'use strict';
 /*
  * Simple BinaryWriter is a minimal tool to write binary stream with unpredictable size.
  * Useful for binary serialization.
  *
- * Copyright (c) 2016 Barbosik
- * 
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- * 
- *     http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * 
- * 
+ * Copyright (c) 2016 Barbosik https://github.com/Barbosik
+ * License: Apache License, Version 2.0
  */
 
-function BinaryWriter(size) {
-    if (!size || size <= 0) {
-        size = Buffer.poolSize / 2;
-    }
-    this._buffer = new Buffer(size);
+function BinaryWriter() {
+    this._buffer = parseInt(process.version[1]) < 6 ? new Buffer(1e5) : Buffer.allocUnsafe(1e5);

```

Comment:
@Andrews54757 

ANDREW STOP, I'M GETTING A SERIOUS HEADACHE. I'M GOING TO SAY IT AGAIN IN PLAIN ENGLISH:

**I WAS FRIENDS WITH THEM, JUST BECAUSE I AM IN A BOTTING DISCORD, DOES NOT MEAN I AM BOTTING. ANDREW DO SOME DAMN RESEARCH, I WAS THERE WAITING FOR INFORMATION ON THEM BOTTING MY WEBSITE. MY WEBSITE ANDREW, NOT OTHER PEOPLE, MY WEBSITE.**

You're actually pissing me off so hard, please simply shut up if you have nothing better to say. Holy shit. I have lost some many brain cells trying to explain one thing to you 12 fucking times. I can't just not swear anymore, I am considering you as brain dead. Either you don't understand plain english or I need to bold it and put it all in caps.

Yes and please DON'T COME BACK. Stop watching this repo because you're trying to waste time for useless additions.